javascript-methods/main.js

```diff
@@ -0,0 +1,44 @@
+var nOne = [2, 4, 8];
+var maximumValue = Math.max(...nOne);
```

Comment:
Please use three variables containing numbers rather than an array, and pass the variables to Math.max individually rather than using the spread operator